core/templates/pages/exploration-player-page/learner-experience/ratings-and-recommendations.component.html

```diff
@@ -0,0 +1,155 @@
+        </oppia-exploration-summary-tile>
+      </div>
+    </div>
+  </div>
+
+  <div *ngIf="inStoryMode" class="conversation-skin-back-to-collection-container">
+    <a [href]="storyViewerUrl" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_RETURN_TO_STORY"></a>
+  </div>
+  <div *ngIf="!collectionSummary && !isRefresherExploration && !inStoryMode" class="conversation-skin-back-to-collection-container">
+    <a href="/community-library" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_RETURN_TO_LIBRARY"></a>
+  </div>
+  <div *ngIf="collectionSummary && !isRefresherExploration" class="conversation-skin-back-to-collection-container">
+    <a [href]="'/collection/' + collectionId" class="conversation-skin-back-to-collection" translate="I18N_PLAYER_BACK_TO_COLLECTION"></a>
+  </div>
+  <div *ngIf="inStoryMode && !userIsLoggedIn" class="conversation-skin-login-container">
+    <div class="conversation-skin-congratulations-text" translate="I18N_CHAPTER_COMPLETION"></div>
+    <span class="conversation-skin-login-text text-secondary" translate="I18N_SAVE_PROGRESS"></span>
+    <div class="mt-2">
+      <button #matButton class="md-raised oppia-login-button text-light protractor-test-login-button" (click)="signIn()">
+        LOGIN
+      </button>
+      <button #matButton class="md-raised oppia-sign-up-button text-light" (click)="signIn()">
+        SIGN UP
+      </button>
+    </div>
+  </div>
+</div>
+
+<style>
+  .conversation-skin-final-summary {
+    margin-bottom: 25px;
+    margin-top: 20px;
+  }
+
+  .conversation-skin-final-ratings-header {
+    color: #064b46;
+    margin: 20px 0 0;
+  }
+
+  .conversation-skin-final-ratings {
+    background-color: white;
+    margin: 0 auto 60px;
+    padding: 11px;
+    text-align: center;
+    width: 38%;
+  }
+
+  .conversation-skin-final-ratings-display {
+    color: #064b46;
+    letter-spacing: 10px;
+  }
+
+  .conversation-skin-final-recommendations {
+    margin-top: 50px;
+    text-align: center;
+  }
+
+  .conversation-skin-final-recommendations-header {
+    color: #064b46;
+    font-size: 150%;
+  }
+
+  .conversation-skin-recommended-activities-container {
+    padding-top: 10px;
+  }
+
+  .oppia-exploration-summary-tile {
+    margin-left: 10px;
+  }
+
+  .conversation-skin-back-to-collection-container {
+    margin-bottom: 92px;
+    margin-top: 25px;
+    text-align: center;
+  }
+
+  .conversation-skin-back-to-collection {
+    color: #064b46;
+    text-decoration: underline;
+  }
+
+  .conversation-skin-login-container {
+    background-color: #FFF;
+    margin: -52px auto 92px;
+    padding: 1.5em 2em;
+    text-align: center;
+    width: 100%;
+  }
+
+  .conversation-skin-login-container .conversation-skin-login-text {
+    font-size: 0.8em;
+  }
+
+  .conversation-skin-login-container .conversation-skin-congratulations-text {
+    font-size: 1.2em;
+    font-weight: 500;
+  }
+
+  .oppia-login-button.md-raised {
```

Comment:
Should this and the next rule follow the .conversation-skin-... pattern, or will this mess up other components?

Reply:
But typically anything that's just prefixed with .oppia is defined in oppia.css and applies to the entire app. Until we change this codebase-wide, I suggest giving this some sort of prefix that clearly scopes it within this component (to other developers) if that is what the intent is.

(I'll also make a note to discuss in the next Angular team meeting what our approach to CSS naming is in light of the Angular migration, so that we have standard patterns to follow (a) during the hybrid phase, (b) post-migration.)